save-frontend/src/main/kotlin/com/saveourtool/save/frontend/utils/RequestUtils.kt

```diff
@@ -682,13 +689,10 @@ private suspend fun Response.inputStream(): Flow<Byte> {
          * available.
          */
         while (true) {
-            @Suppress(
-                "GENERIC_VARIABLE_WRONG_DECLARATION",
-                "TYPE_ALIAS",
-            )
+            @Suppress("GENERIC_VARIABLE_WRONG_DECLARATION")
```

Comment:
```suggestion
            val resultAsync: ResultAsync<Uint8Array> = reader
```
does it resolve suppression?

Reply:
Yes it does, thanks.

Addressed with 7fd1bd570113cb3581186b6bcf497f088ac1f418